<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Billet_MeasurementClass</class>
 <widget class="QMainWindow" name="Billet_MeasurementClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>钢坏轮廓在线测量系统   </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>390</width>
      <height>260</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_left_up">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>390</width>
       <height>260</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1626</width>
       <height>1236</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>270</y>
      <width>390</width>
      <height>260</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_left_down">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>390</width>
       <height>260</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1626</width>
       <height>1236</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>0</y>
      <width>390</width>
      <height>260</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_right_up">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>390</width>
       <height>260</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1626</width>
       <height>1236</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>270</y>
      <width>390</width>
      <height>260</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_right_down">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>390</width>
       <height>260</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1626</width>
       <height>1236</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_connect"/>
   <addaction name="separator"/>
   <addaction name="action_disconnect"/>
   <addaction name="separator"/>
   <addaction name="action_start"/>
   <addaction name="separator"/>
   <addaction name="action_pause"/>
   <addaction name="separator"/>
   <addaction name="action_setcampara"/>
   <addaction name="separator"/>
   <addaction name="action_ExTriggerON"/>
   <addaction name="separator"/>
   <addaction name="action_ExTriggerOff"/>
   <addaction name="separator"/>
   <addaction name="action_TriggerParam"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于我们</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="action_connect">
   <property name="icon">
    <iconset resource="billet_measurement.qrc">
     <normaloff>:/image/image/connect.ico</normaloff>:/image/image/connect.ico</iconset>
   </property>
   <property name="text">
    <string>连接相机</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="icon">
    <iconset resource="billet_measurement.qrc">
     <normaloff>:/image/image/disconnect.ico</normaloff>:/image/image/disconnect.ico</iconset>
   </property>
   <property name="text">
    <string>断开相机</string>
   </property>
  </action>
  <action name="action_start">
   <property name="icon">
    <iconset resource="billet_measurement.qrc">
     <normaloff>:/image/image/start.ico</normaloff>:/image/image/start.ico</iconset>
   </property>
   <property name="text">
    <string>开始采集</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="icon">
    <iconset resource="billet_measurement.qrc">
     <normaloff>:/image/image/pasue.ico</normaloff>:/image/image/pasue.ico</iconset>
   </property>
   <property name="text">
    <string>停止采集</string>
   </property>
  </action>
  <action name="action_setcampara">
   <property name="icon">
    <iconset resource="billet_measurement.qrc">
     <normaloff>:/image/image/Setting_32px.ico</normaloff>:/image/image/Setting_32px.ico</iconset>
   </property>
   <property name="text">
    <string>设置相机参数</string>
   </property>
  </action>
  <action name="action_ExTriggerON">
   <property name="icon">
    <iconset resource="billet_measurement.qrc">
     <normaloff>:/image/image/开关 开(1).png</normaloff>:/image/image/开关 开(1).png</iconset>
   </property>
   <property name="text">
    <string>打开外部触发</string>
   </property>
  </action>
  <action name="action_ExTriggerOff">
   <property name="icon">
    <iconset resource="billet_measurement.qrc">
     <normaloff>:/image/image/开关 关(1).png</normaloff>:/image/image/开关 关(1).png</iconset>
   </property>
   <property name="text">
    <string>关闭外部触发</string>
   </property>
  </action>
  <action name="action_TriggerParam">
   <property name="icon">
    <iconset resource="billet_measurement.qrc">
     <normaloff>:/image/image/设置参数.png</normaloff>:/image/image/设置参数.png</iconset>
   </property>
   <property name="text">
    <string>设置外部触发参数</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="billet_measurement.qrc"/>
 </resources>
 <connections/>
</ui>
